.circleci/real_config.yml

```diff
@@ -118,6 +118,12 @@ commands:
     steps:
       - run: echo 'CIRCLE_WORKING_DIRECTORY="${CIRCLE_WORKING_DIRECTORY/#\~/$HOME}"' >> $BASH_ENV
 
+  python-report:
+    description: "Help with debugging Python and pip related issues."
+    steps:
+      # keep it a single line to avoid adding ci overhead. each step is about 5s.
+      - run: python --version; pip --version; pip list; sh -c "pip check || true"
```

Comment:
can you try using block scalar syntax with `>`?
https://yaml-multiline.info/

Reply:
like this
```
- run: |
    git add --all
    git commit -am "a commit message"
    git push
```

Reply:
👍🏼 and "single step" would be more accurate for the comment

Reply:
id be cool with removing the comment altogether. let the syntax serve as an example. either way is cool